generators/node/index.js

```diff
@@ -0,0 +1,498 @@
+'use strict'
+const path = require('path')
+const {default: chalk} = require('chalk')
+const yosay = require('yosay')
+const inquirer = require('inquirer')
+const Generator = require('../../lib/generator')
+const validator = require('./validator')
+
+module.exports = class extends Generator {
+  constructor(args, opts) {
+    super(args, opts)
+
+    this.option('skip-welcome', {
```

Comment:
This option and its use below is the only actual change I made to this file (which was renamed from `generators/app.js`).